cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -162,9 +166,28 @@ func VerifyBlobCmd(ctx context.Context, ko sign.KeyOpts, certRef, certEmail, cer
 		}
 	}
 
-	// verify the signature
-	if err := verifier.VerifySignature(bytes.NewReader([]byte(sig)), bytes.NewReader(blobBytes)); err != nil {
-		return err
+	// Is DSSE?
+	DSSEenvelope := ssldsse.Envelope{}
+	if err := json.Unmarshal(blobBytes, &DSSEenvelope); err == nil {
```

Comment:
Could we move this into a separate function and add a unit test for it?

Reply:
Ok, I found some help in the sigstore package, no the code is shorter and cleaner.